### DISCUSSION TITLE
how to integrate 12 step meeting finder in react js

### DISCUSSION THREAD

**Discussion by patelsaurav** [code4recovery/12-step-meeting-list#1065] [Q&A]
i need to display all the 12 step meeting list that happen near by user, so plese refer any doc  for guidence..

**Comment by joshreisner** [comment]
hi, we do have a react-based meeting finder called TSML UI

in our FAQ we do have items on:
* [enable "Near Me" mode](https://github.com/code4recovery/tsml-ui#enable-near-me-mode)
* [importing into react applications](https://github.com/code4recovery/tsml-ui#can-i-import-tsml-ui-from-npm-for-use-in-a-nextjs-or-gatsby-project) (it's not possible to do that yet)

instructions on how to embed it via HTML are here: https://tsml-ui.code4recovery.org/
